config/initializers/omniauth.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'], skip_jwt: true
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'], {scope: "userinfo.email, calendar", skip_jwt: true}
```

Comment:
Add the `calendar` scope to the google oauth and this slightly changes the response from google. The response with this scope does not include the `name` of the `user`. Could we have the `user` enter this instead?